ngraph/frontend/onnx_import/src/op/round.cpp

```diff
@@ -30,26 +30,12 @@ namespace ngraph
         {
             namespace set_1
             {
-                // WARNING!
-                // Current version is:
-                // data_floor = floor(data)
-                // diff = data - data_floor
-                // if(diff < 0.5f)
-                //   return data_floor
-                // else
-                //   return data_floor + 1.0f
-                //
-                // The correct version should contain condition:
-                // if (diff < 0.5f || (diff == 0.5f && static_cast<int>(data_floor) % 2 == 0))
                 OutputVector round(const Node& node)
                 {
-                    const Output<ngraph::Node> data{node.get_ng_inputs().at(0)};
-                    const auto half_const =
-                        default_opset::Constant::create(data.get_element_type(), {}, {0.5f});
-
-                    // Floor(data + 0.5)
-                    return {std::make_shared<default_opset::Floor>(
-                        std::make_shared<default_opset::Add>(data, half_const))};
+                    return {std::make_shared<default_opset::Round>(
+                        node.get_ng_inputs().at(0),
+                        static_cast<default_opset::Round::RoundMode>(
+                            node.get_attribute_value<int64_t>("mode", 0)))};
```

Comment:
There is no "mode" attribute in the [ONNX Round](https://github.com/onnx/onnx/blob/master/docs/Changelog.md#Round-11) doc and the rule seems to be always `RoundMode:::HALF_TO_EVEN`

Reply:
It's from our documentation: https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/arithmetic/Round_5.md

> Operation takes one input tensor and rounds the values, element-wise, meaning it finds the nearest integer for each value. In case of halfs, the rule is to round them to the nearest even integer if mode attribute is half_to_even or rounding in such a way that the result heads away from zero if mode attribute is half_away_from_zero.

Reply:
I understand it can be used in some custom onnx models.